website_app/app.py

```diff
@@ -0,0 +1,68 @@
+from flask import Flask, render_template, request, url_for, flash, redirect
+from werkzeug.exceptions import abort
+import scrape, graph
+import sentiment_analysis as sen
+
+app = Flask(__name__)
+app.config['SECRET_KEY'] = 'admin123_foobar'
+
+def main_process(submit_data):
+    keyword, start_date, end_date, output_tag = submit_data
+    polarity_tags = ["Tweet", "Polarity", "Subjectivity"]
+    
+    scraped_data_folder = "scraped_data/"
+    full_polarity_folder = "processed_data/"
+    average_folder = "averaged_data/"
+    images_folder = "data_images/"
+
+    
+    scraped_output_file = scraped_data_folder + output_tag + "_scraped.json"
+    full_polarity_file = full_polarity_folder + output_tag + "_pol.json"
+    avg_polarity_file = average_folder + output_tag + "_avg.json"
+
+    scrape.scrape_twitter(keyword, start_date, end_date, scraped_output_file)
+    
+    sen.main_process_polarity(scraped_output_file, 
+                        polarity_tags, full_polarity_file, avg_polarity_file)
+    
+    # currently using the full_polarity file
+    graph.visualize_data(full_polarity_file, images_folder + output_tag)
+    
+
+
+
+
+@app.route('/', methods=('GET', 'POST'))
+def index():
+    # alter this code to get the keyword submission
+    # include a way to access previous keyword searches
+    print("hi")
+    if request.method == 'POST':
+        
+        keyword = request.form['keyword']
+        start_date = request.form['start_date']
+        end_date = request.form['end_date']
+        output_tag = "dummy_name.json" # try to make this have a unique id fo every process
```

Comment:
I think this should just be "dummy_name" ? since we're appending .json inside main_process